chart/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
```

Comment:
Consider adding an icon field.

It is optional but it does throw a warning that it is missing if you run `helm lint`

I'm not sure what the most appropriate icon would be for llama-stack.  The project doesn't appear to have an official icon as far as I can tell.